plugins/node/opentelemetry-instrumentation-winston/README.md

```diff
@@ -19,47 +19,93 @@ npm install --save @opentelemetry/instrumentation-winston
 
 ```js
 const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
+const logsAPI = require('@opentelemetry/api-logs');
+const {
+    LoggerProvider,
+    SimpleLogRecordProcessor,
+    ConsoleLogRecordExporter,
+} = require('@opentelemetry/sdk-logs');
 const { WinstonInstrumentation } = require('@opentelemetry/instrumentation-winston');
 const { registerInstrumentations } = require('@opentelemetry/instrumentation');
 
-const provider = new NodeTracerProvider();
-provider.register();
+const tracerProvider = new NodeTracerProvider();
+tracerProvider.register();
+
+// To start a logger, you first need to initialize the Logger provider.
+const loggerProvider = new LoggerProvider();
+// Add a processor to export log record
+loggerProvider.addLogRecordProcessor(
+    new SimpleLogRecordProcessor(new ConsoleLogRecordExporter())
+);
+logsAPI.logs.setGlobalLoggerProvider(loggerProvider);
 
 registerInstrumentations({
-  instrumentations: [
-    new WinstonInstrumentation({
-      // Optional hook to insert additional context to log metadata.
-      // Called after trace context is injected to metadata.
-      logHook: (span, record) => {
-        record['resource.service.name'] = provider.resource.attributes['service.name'];
-      },
-    }),
-    // other instrumentations
-  ],
+    instrumentations: [
+        new WinstonInstrumentation({
+            // See below for Winston instrumentation options.
+        }),
+    ],
 });
 
 const winston = require('winston');
 const logger = winston.createLogger({
-  transports: [new winston.transports.Console()],
+    transports: [new winston.transports.Console()],
 })
 logger.info('foobar');
 // {"message":"foobar","trace_id":"e21c7a95fff34e04f77c7bd518779621","span_id":"b7589a981fde09f4","trace_flags":"01", ...}
 ```
 
-### Fields added to Winston metadata
+### Winston instrumentation options
+
+| Option                  | Type              | Description |
+| ----------------------- | ----------------- | ----------- |
+| `enableLogSending`      | `boolean`         | Whether to enable [log sending](#log-sending). Default `false`. |
```

Comment:
See @Flarna's latest comment on https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1837#discussion_r1474080114
Perhaps we are back to `disableLogSending`?

Reply:
Well then original concern is back again right? we will log an error in our diag Api if transport package is not present, @Flarna thoughts?

Reply:
I see @Flarna comment now, to much stuff happening here :) I can update to disableLogSending defaulting to false same as Bunyan, if everyone is ok with that, the error will logged if transport is not present though

Reply:
Feel free to go for enabled on default to stay consistent with bunyan.
Regardind log: In my opinion info or maybe warning log level would be better but error is also acceptable for me.

Reply:
FWIW, I would be fine having it be off by default. (I'd be fine either way.) The OTel docs on logging and the OTel Java SDK config docs make it clear that it is a typical user's choice on whether they want to (a) log to stdout/file with added trace_id et al fields and have a collector ingest them *or* (b) have the SDK directly send logs to some OTLP endpoint.  So that argues for not having both facilities on by default.